sensor-system/migrations/v0.2.sql

```diff
@@ -0,0 +1,21 @@
+CREATE TABLE Location (
+    id SERIAL PRIMARY KEY,
+    name VARCHAR(32) NOT NULL,
+    address VARCHAR(128) NOT NULL
+);
+
+CREATE TABLE Device (
+    id SERIAL PRIMARY KEY,
+    name VARCHAR(32) NOT NULL,
+    location_id INTEGER NOT NULL,
+    status VARCHAR(16) DEFAULT 'active',
+    FOREIGN KEY (location_id) REFERENCES Location(id) ON DELETE CASCADE
+);
+
+CREATE TABLE Count (
+    id SERIAL PRIMARY KEY,
+    device_id INT NOT NULL,
+    timestamp TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    count_value INT NOT NULL,
+    FOREIGN KEY (device_id) REFERENCES Device(id) ON DELETE CASCADE
```

Comment:
```suggestion
CREATE TABLE Count (
    "id" SERIAL PRIMARY KEY,
    "deviceId" INT NOT NULL,
    "timestamp" TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
    "in" INT NOT NULL,
    "out" INT NOT NULL,
    FOREIGN KEY (deviceId) REFERENCES Device(id) ON DELETE CASCADE
```

Count packets will push every minute containing both in and out values. Your count table should have an in and out column to track this. 

Timestamp is a postgres keyword, once again it's easier to just wrap every column def in quotes